src/pushover_complete/pushover_api.py

```diff
@@ -21,6 +21,9 @@ class PushoverAPI(object):
     def __init__(self, token):
         self.token = token
 
+    def __repr__(self):
+        return "PushoverAPI(token={})".format(self.token)
```

Comment:
gotta add some tests for this. Are these standard `__repr__` functions suppsoed to get the class name from `self.__class__` instead of being hard-coded?